### PR TITLE
March 4th Change Log

### PR DESCRIPTION
**Pull request by soutenniza**
## Changes proposed in this pull request:
- Change log!

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/march4)

## Security Considerations
None

### CODE REVIEW

File: _posts/2022-03-04-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
+---
+layout: post
+date: March 4th 2022
+title: "March 4th cloud.gov Change Log"
+excerpt: The cloud.gov team is working on providing change logs so everyone can see new features and updates.
+---
+
+Happy Women's History Month! The cloud.gov team is working on providing change logs so everyone can see new features and updates. Highlights this month include Cloud Foundry Deployment and AWS RDS Broker updates.
```
Comment (apburnes) [inline _posts/2022-03-04-release-notes.md:8]: Nice touch 👍 .

File: _posts/2022-03-04-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+### CF Networking Release - v3.0.0 up from v2.42.0
+
+Release Highlights
+* ✨ [New Feature] cf-networking + silk-release now support dynamically updating ASG data for app containers without needing a restart!

+* A new job policy-server-asg-syncer queries CAPI for ASG data periodically and updates the policy-server database
+* A new endpoint on policy-server-internal is exposed to allow vxlan-policy-agent to query for ASG data
+* When disabled, everything behaves as it did previously. 
 - To disable, set the disable property of policy-server-asg-syncer to true
```
Comment (apburnes) [inline _posts/2022-03-04-release-notes.md]:
```suggestion
* When disabled, everything behaves as it did previously.  - To disable, set the disable property of policy-server-asg-syncer to true
```

File: _posts/2022-03-04-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+
+BBS
+* cloudfoundry/bbs@f246cdd Update LRP instance when internal routes were changed
+* cloudfoundry/bbs@83003cf Update readme with latest steps to generate ruby protobuf files
+* cloudfoundry/bbs@e56feb1 Fix instructions for building ruby protobuf files* 
+
+### CF Networking Release - v3.0.0 up from v2.42.0
+
+Release Highlights
+* ✨ [New Feature] cf-networking + silk-release now support dynamically updating ASG data for app containers without needing a restart!
```
Comment (apburnes) [inline _posts/2022-03-04-release-notes.md:148]:
```suggestion
* ✨ [New Feature] cf-networking + silk-release now support dynamically updating ASG data for app containers without needing a restart!
```

Comment (apburnes) [reply _posts/2022-03-04-release-notes.md:148]: Removes empty box artifact.
Screen Shot 2022-03-04 at 9 15 00 AM

File: _posts/2022-03-04-release-notes.md (hunk 2 above)
Comment (apburnes) [reply _posts/2022-03-04-release-notes.md]: Removes an empty box artifact.
Screen Shot 2022-03-04 at 9 13 54 AM

File: _posts/2022-03-04-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+### Credhub Release - 2.12.0 up from 2.11.2
+
+Security Fixes
+* Bump various dependencies.
+
+Bug Fixes
+* Fixes an issue where CredHub experiences downtime during certificate rotation process by making CredHub properly load concatenated mTLS CA certificates.
+
+Features
+* CredHub is now compatible with Postgres 13, 14.
```
Comment (bengerman13) [inline _posts/2022-03-04-release-notes.md:206]: I wonder if we should skip this, just so we don't confuse people?

Comment (soutenniza) [reply _posts/2022-03-04-release-notes.md:206]: Thats why it's hidden into platform changes? Would adding internal use only be useful?

Comment (bengerman13) [reply _posts/2022-03-04-release-notes.md:206]: I suppose it's fine - I missed the platform heading 🤦

**Review by apburnes — APPROVED** [review]
👍

**Pull request merged**